ci/ray_ci/tester.py

```diff
@@ -352,7 +353,18 @@ def _get_flaky_test_targets(
         yaml_dir = os.path.join(bazel_workspace_dir, "ci/ray_ci")
 
     with open(f"{yaml_dir}/{team}.tests.yml", "rb") as f:
+        # load flaky tests from yaml
         all_flaky_tests = yaml.safe_load(f)["flaky_tests"]
+        # load flaky tests from DB
+        all_flaky_tests.extend(
```

Comment:
do we want to deduplicate (and sort)? what if both the yaml and the db contains the same test?

and could you add a test for that case?